tests/integration/features/sign/request.feature

```diff
@@ -113,23 +113,25 @@ Feature: request-signature
     Given as user "admin"
     And my inbox is empty
     And run the command "libresign:developer:reset --all"
-    And run the command "config:app:set libresign renewal_interval --value 1"
-    When sending "post" to ocs "/apps/libresign/api/v1/request-signature"
+    And sending "post" to ocs "/apps/libresign/api/v1/request-signature"
       | file | {"url":"<BASE_URL>/apps/libresign/develop/pdf"} |
       | users | [{"identify":{"email":"signer2@domain.test"}}] |
       | name | document |
-    Then the response should have a status code 200
+    And the response should have a status code 200
```

Comment:
In this case which is the difference of "Then" and "And" usage ? Maybe "Then" make more sense...

Reply:
I used `And` because is part of a background scenario but if I put as  `background scenario`, this will run for each `scenario` in this `feature`.

I only used `Then` when is the objective of action that we made after a `When`

Given: preparing the screnario
When: The start of action that we need to check by Then
Then: The objective of scenario

Reply:
I made a fix in a point that was `And` and was best to be `When`